src/python/pants/backend/python/rules/pex.py

```diff
@@ -407,12 +381,14 @@ async def create_pex(
         ),
     )
 
-    if pex_debug.might_log:
+    if verbosity > 0:
         lines = result.stderr.decode().splitlines()
+
         if lines:
-            pex_debug.log(f"Debug output from Pex for: {process}")
+            log_level = LogLevel.DEBUG if verbosity <= 3 else LogLevel.TRACE
```

Comment:
Ah, so this would require that you both set the `--pex` option and `-ldebug` or `ltrace`. Instead, if `--pex-verbosity` is set, I suspect we should _always_ log Pex output. So, maybe use `LogLevel.INFO` unconditionally. Thoughts?

Reply:
I don't have a preference here. Having PEX debug output emitted as INFO seems weird though.

Reply:
It is indeed weird, but I only expect people to set `--pex-verbosity` on the CLI as a short term thing to debug, so I think the weirdness is justified.

Otherwise, users would have to use both `./pants -ldebug --pex-verbosity=n` and deal with all the super verbose debug messages from the rest of Pants.

Reply:
I set it to INFO level.